R/configure.R

```diff
@@ -1,13 +1,34 @@
 ##' Configure your hipercow root.  `hipercow_configure` creates the
-##' configuration and `hipercow_configuration` looks it up.
+##' configuration and `hipercow_configuration` looks it up
+##'
+##' # Windows
+##'
+##' Options supported by the `windows` driver:
+##'
+##' * `shares`: Information about shares (additional to the one
+##'   mounted as your working directory) that should be made available
+##'   to the cluster job. The use case here is where you need access
+##'   to some files that are present on a shared drive and you will
+##'   access these by absolute path (say `M:/gis/shapefiles/`) from
+##'   your tasks.  You can provide a share as a `windows_path` object,
+##'   or a list of such objects.  You will not typically need to use
+##'   this option.
+##'
+##' * `r_version`: Control the R version used on the
+##'   cluster. Typically hipercow will choose a version close to the
+##'   one you are using to submit jobs, of the set available on the
+##'   cluster. You can use this option to choose a specific version
+##'   (e.g., pass "4.3.0" to select exactly that version).
+##'
+##' See `vignette("details")` for more information about these options.
```

Comment:
To check - does the vignette include which specific R versions are available / where else is that information.

Reply:
It has a hard coded list for now, but I've made mrc-4927 to solve the general problem of reporting this.